plutus-core/untyped-plutus-core/test/Transform/inlinePure3.plc.golden

```diff
@@ -0,0 +1 @@
+(lam b_1 [ (lam x_4 (lam y_5 [ x_4 x_4 ])) (con integer 1) ])
```

Comment:
should something have been inlined here?

Reply:
No, and this is deliberate. It isn't pre-inlined because `x_4` is used twice. And it isn't post-inlined because `sizeIsAcceptable (Constant {}) = False`.

Reply:
Maybe you should include a comment, then? Otherwise it's not clear what the intended behaviour is. TBH they could all have comments.